xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -34,13 +34,19 @@
 </div>
 #if ($editor != 'inline')
 <div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+    <dt>
+      <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
+    </dt>
+    <dd>
+      <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+    </dd>
 </div>
 #end
 
 #if($editor == 'wiki')
 <div id="contentMeta">
-  <label for="content">$services.localization.render('core.editors.content.contentField.label')</label>
+    <dt>
+      <label for="content">$services.localization.render('core.editors.content.contentField.label')</label>
+    </dt>
```

Comment:
Where is the ``dd`` that corresponds to this definition term?

Reply:
A definition term (``dt``) should be followed by a definition (``dd``). If there's no definition then the ``dd`` remains empty.

Reply:
Ok got it.